demo/fiber/skynet_test/src/main/java/com/example/Skynet.java

```diff
@@ -0,0 +1,113 @@
+package com.example;/*
```

Comment:
Generally, the package declaration comes after the copyright not.

Reply:
Thanks for your reviews, I'll update the relevant content soon.